chain/client/src/test_utils.rs

```diff
@@ -197,11 +200,15 @@ pub fn setup(
     transaction_validity_period: NumBlocks,
     genesis_time: DateTime<Utc>,
     ctx: &Context<ClientActor>,
-) -> (Block, ClientActor, Addr<ViewClientActor>) {
+) -> (Block, ClientActor, Addr<ViewClientActor>, Arc<dyn ShardsManagerAdapter>) {
```

Comment:
Can we use ActorHandlesForNetworkMessageMocking here as well?

Reply:
Hmm, that struct contains a Addr<ClientActor> instead of ClientActor.

Reply:
The way we start actors here is inconsistent from the way we start actors elsewhere. I'm not keen on touching this though, it might break stuff.